core/src/test/scala/org/apache/spark/internal/config/ConfigEntrySuite.scala

```diff
@@ -51,6 +52,26 @@ class ConfigEntrySuite extends SparkFunSuite {
     assert(conf.get(dConf) === 20.0)
   }
 
+  test("conf entry: timezone") {
+    val tzStart = TimeZone.getDefault().getID()
```

Comment:
@cloud-fan Thank you :-). Its so much simpler. I somehow was trying to use the exact same function that we use in the code.